sageodata-apispec.yml

```diff
@@ -2351,21 +2352,82 @@ components:
       properties:
         unitNumber:
           format: int32
+          description: A combination of the map100000Number and sequence number.
           type: integer
         nrmRegionCode:
           description: NRM region code
+          enum:
+          - Eyre
+          - Peninsula
+          - South
+          - Australian
+          - Arid
+          - Lands
+          - East
+          - Northern
+          - '&'
+          - Yorke
+          - Alinytjara
+          - Wilurara
+          - Adelaide
+          - Mt
+          - Lofty
+          - Ranges
+          - Murray-Darling
+          - Basin
+          - Kangaroo
+          - Island
```

Comment:
Each word in here has now been defined as a valid enum value. For eg: 

> Mt, 
> Basin etc.

I would recommend using a key as the enum value and provide a description as to what it signifies.
An example would be the _testMethod_ variable in the _wcr-pumptestdata_

Reply:
SA Geodata currently has the following data in _DD_NRM_REGION_ table:
|Code|Value|
|---|---|
|Adelaide & Mt Lofty Ranges	|	Adelaide & Mt Lofty Ranges|
|Alinytjara Wilurara	|	Alinytjara Wilurara|
|Eyre Peninsula	|	Eyre Peninsula|
|Kangaroo Island	|	Kangaroo Island|
|Northern & Yorke	|	Northern & Yorke|
|South Australian Arid Lands	|	South Australian Arid Lands|
|South East	|	South East|
|South Australian Murray-Darling Basin	|	South Australian Murray-Darling Basin|
|out of NRM regions	|	out of NRM regions|